src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -150,31 +160,36 @@ public void addPerson(Person person) {
     public void addCompany(Company company) {
         addressBook.addCompany(company);
         updateFilteredCompanyList(PREDICATE_SHOW_ALL_COMPANIES);
+        context = Context.COMPANY;
     }
+
     @Override
     public void addJob(Job job) {
         addressBook.addJob(job);
         updateFilteredJobList(PREDICATE_SHOW_ALL_JOBS);
+        context = Context.JOB;
     }
 
     @Override
     public void setPerson(Person target, Person editedPerson) {
         requireAllNonNull(target, editedPerson);
 
         addressBook.setPerson(target, editedPerson);
+        context = Context.CONTACT;
     }
 
     /**
      * Replaces the target company with an edited version of itself.
      *
-     * @param target Company to be replaced.
+     * @param target        Company to be replaced.
```

Comment:
What's this supposed to be? If it's a tab I think you should remove it.

Reply:
![image](https://github.com/user-attachments/assets/1a76d051-ff85-4780-8d0b-e5785c817735)

It is the auto reformat in my IDE to align parameter description. I don't think it violates any coding standards, but I am more than happy disable that setting, just let me know.

Reply:
I guess it's fine as long as the spacing isn't unintentional, feel free to resolve the two convos